core/common/src/thrift/file_system_master.thrift

```diff
@@ -123,6 +123,27 @@ struct MountTOptions {
 }
 struct MountTResponse {}
 
+struct GetMountTableTResponse {}
+
+/*
+struct GetMountTableTResponse {
+  1: map<string, MountPointInfo> mountTable
+}
+
+struct MountPointInfo {
```

Comment:
I wonder if we need to make changes in RetryHandlingFileSystemMasterClient.java.